bundle/permissions/workspace_root.go

```diff
@@ -77,3 +84,32 @@ func getWorkspaceObjectPermissionLevel(bundlePermission string) (workspace.Works
 		return "", fmt.Errorf("unsupported bundle permission level %s", bundlePermission)
 	}
 }
+
+// checkWorkspaceRootPermissions checks that if permissions are set for the workspace root, and workspace root starts with /Workspace/Shared, then permissions should be set for group: users
+func checkWorkspaceRootPermissions(b *bundle.Bundle) diag.Diagnostics {
+	var diags diag.Diagnostics
+	if len(b.Config.Permissions) == 0 {
```

Comment:
We should also warn if you're not explicitly configuring permissions and are using `/Shared`.